powerhome/playbook
# [NUXE-783] Walkthrough Kit

#### Screens

![Screen Recording 2021-06-30 at 12 42 26 PM](https://user-images.githubusercontent.com/9784112/123999681-c6e7b200-d9a0-11eb-81de-ad05390c2dd6.gif)

This PR adds the walkthrough kit to Playbook. It allows users to create guided tours and/or hotspots that provide information


#### Breaking Changes
No this is a new kit

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUXE-783

#### How to test this

Click Start Tour and/or a red Beacon to start the tour
After cycling through all steps 
The tour should end/ disappear and no more beacons should be on the screen 

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
